docs/js-quickstart.md

```diff
@@ -5,56 +5,51 @@ sidebar_label: Getting Started with JavaScript Testing
 description: Basic steps for getting going quickly with JavaScript based frameworks using TestRunner Toolkit and SauceCTL
 ---
 
-## Step 1: Install SauceCTL and the TestRunner Toolkit
+## Step 1: Install sauceCTL and the TestRunner Toolkit
 
-You can configure the [Sauce Testrunner Toolkit](https://docs.saucelabs.com/testrunner-toolkit) along with Docker, _or_ run [`saucectl`](https://docs.saucelabs.com/testrunner-toolkit/saucectl) on Sauce Labs VMs. 
+You can configure the [Sauce Testrunner Toolkit](https://docs.saucelabs.com/testrunner-toolkit) along with Docker, _or_ run [`saucectl`](https://docs.saucelabs.com/testrunner-toolkit/saucectl) on Sauce Labs VMs.
 
 The basic steps include:
 
- * Install SauceCTL with `npm i -g saucectl`
+*   Install saucectl with `npm i -g saucectl`
+*   Set your Sauce username and access key with `saucectl configure`
+*   Download an example test suite
+    *   [Cypress](https://github.com/saucelabs/saucectl-cypress-example)
+    *   [TestCafe](https://github.com/saucelabs/saucectl-testcafe-example)
+    *   [Playwright](https://github.com/saucelabs/saucectl-playwright-example)
+    *   [Puppeteer](https://github.com/saucelabs/saucectl-puppeteer-example)
+*   If you are ready to use your own test
+    *   Add `.sauceignore `, `.sauce `and `.sauce/config.yml` for your framework using examples above, to your project
+    *   Update `config.yml` with test suite information including location and dependencies
 
- * Set your Sauce username and access key with `saucectl configure`
+ > For more detailed instructions please [consult the documentation](https://docs.saucelabs.com/testrunner-toolkit/installation), or see [the Cypress on Sauce Course](https://training.saucelabs.com/saucectl/)
 
- * Create a new SauceCTL project with `saucectl new`
 
- * Update `config.yml` with test suite information including location and dependencies
+## Step 2: Run Your Tests
 
- > For more detailed instructions please [consult the documentation](https://docs.saucelabs.com/testrunner-toolkit/installation), or see [an example with Cypress ](https://training.saucelabs.com/codelabs/Module1-Testrunner/index.html?index=..%2F..testrunner#2)
+By default your Cypress, TestCafe, or Playwright tests test suites are run on Sauce Labs VMs when you run the command:
 
+### `saucectl run`
```

Comment:
Okay. If you feel strongly about including the modes, I won't push back, but I would like to change a few things about these intro lines:
1. Could you please rewrite the sentence so it doesn't read "tests test suites"?
2. Can we make `saucectl run` a code box instead of a heading? It is awkward to have a colon introduce a new heading, grammatically, plus I think it is a little confusing to have the command itself represented as a heading.

Reply:
done